h2o-r/tests/testdir_jira/runit_pubdev2801.R

```diff
@@ -0,0 +1,12 @@
+setwd(normalizePath(dirname(R.utils::commandArgs(asValues=TRUE)$"f")))
+source("../../scripts/h2o-r-test-setup.R")
+
+#Starting H2O server from R ignores IP and port parameters
+#This is a test to see if there is a proper fix in place
+
+test_pubdev2801 <- function(){
+  h2o.init(port = 54355)
```

Comment:
Yeah, just to confirm it is actually started on 54355. It could eventually ignore port argument and 54321, then properly shutdown, and we wouldn't spot regression.